terraform/core/38-api-ingestion.tf

```diff
@@ -53,7 +54,7 @@ module "vonage_api_ingestion" {
     "SECRET_NAME"           = "/customer-services/vonage-key"
     "TARGET_S3_BUCKET_NAME" = local.s3_target_bucket_name
     "OUTPUT_FOLDER"         = "customer-services/manual/vonage"
-    "TRIGGER_NAME" = "TRIGGER_NAME"
+    "TRIGGER_NAME" = local.vonage_glue_trigger_name
     "API_TO_CALL" = "stats"
     "TABLE_TO_CALL" = "interactions"
```

Comment:
These should all align at the =, there might be a formatter you can get for pycharm?